arcgis-ios-sdk-samples/Layers/Display annotation/README.metadata.json

```
@@ -0,0 +1,26 @@
+{
+    "category": "Layers",
+    "description": "Display annotation from a feature service URL.",
+    "ignore": false,
+    "images": [
+        "DisplayAnnotation.png"
+    ],
+    "keywords": [
+        "Annotation",
+        "Cartography",
+        "Labels",
+        "Placement",
+        "Reference scale",
+        "Text",
+        "Utility"
+    ],
+    "redirect_from": [],
+    "relevant_apis": [
+        "AGSAnnotationLayer",
+        "AGSFeatureLayer"
+    ],
+    "snippets": [
+        "DisplayAnnotationViewController.swift"
+    ],
+    "title": "Display Annotation"
```

Capitalization does matter here. Esri's style is to only capitalize the first letter of the first word in a title, so this should be "Display annotation". 

```suggestion
    "title": "Display annotation"
```

That way it matches the capitalization of the other sample titles on the [developers website](https://developers.arcgis.com/ios/latest/swift/sample-code/). It looks like we weren't entirely consistent ("Symbolize Shapefile" should be "Symbolize shapefile", for example), but we tried our best.